### DISCUSSION TITLE
Save thumbnail from active directory in pocketbase

### DISCUSSION THREAD

**Discussion by starredev** [pocketbase/pocketbase#2441] [Q&A]
I've a question.

How to store thumbnail foto into pocketbase? I tried to convert base64 but I receive some errors from pocketbase. "Failed to submitting data due invalid format"

```thumbnailPhoto: '����\x00\x10JFIF\x00\x01\x01\x01\x00`\x00`\x00\x00��\x00C\x00\b\x06\x06\x07\x06\x05\b\x07\x07\x07\t\t\b\n' +
    '\f\x14\r\f\x0B\x0B\f\x19\x12\x13\x0F\x14\x1D\x1A\x1F\x1E\x1D\x1A\x1C\x1C $.\' ",#\x1C\x1C(7),01444\x1F\'9=82<.342��\x00C\x01\t\t\t\f\x0B\f\x18\r\r\x182!\x1C!22222222222222222222222222222222222222222222222222��\x00\x11\b\x00`\x00V\x03\x01"\x00\x02\x11\x01\x03\x11\x01��\x00\x1F\x00\x00\x01\x05\x01\x01\x01\x01\x01\x01\x00\x00\x00\x00\x00\x00\x00\x00\x01\x02\x03\x04\x05\x06\x07\b\t\n' +
    '\x0B��\x00�\x10\x00\x02\x01\x03\x03\x02\x04\x03\x05\x05\x04\x04\x00\x00\x01}\x01\x02\x03\x00\x04\x11\x05\x12!1A\x06\x13Qa\x07"q\x142���\b#B��\x15R��$3br�\t\n' +
    "\x16\x17\x18\x19\x1A%&'()*456789:CDEFGHIJSTUVWXYZcdefghijstuvwxyz���������������������������������������������������������������������������\x00\x1F\x01\x00\x03\x01\x01\x01\x01\x01\x01\x01\x01\x01\x00\x00\x00\x00\x00\x00\x01\x02\x03\x04\x05\x06\x07\b\t\n" +
    '\x0B��\x00�\x11\x00\x02\x01\x02\x04\x04\x03\x04\x07\x05\x04\x04\x00\x01\x02w\x00\x01\x02\x03\x11\x04\x05!1\x06\x12AQ\x07aq\x13"2�\b\x14B����\t#3R�\x15br�\n' +
    '\x16$4�%�\x17\x18\x19\x1A&\'()*56789:CDEFGHIJSTUVWXYZcdefghijstuvwxyz��������������������������������������������������������������������������\x00\f\x03\x01\x00\x02\x11\x03\x11\x00?\x00���\x0B��+��یy��\x1E��\x1Dj4��\tpE��\x07\x1CHpG�Wo좏S�i�\x01|�%sך��4�\\\x14@�\x06\x01\x1D��\x16��S�oY����H�\x0Fִ#MC;��\'Q�\tO?�cͯ����R\x0E3ީ=����r\x07Q�T�as��z[.\r�����RsU��l���M.=K����<�J\\�nڝ�I�B��d�\x1A�\x1E)�d��S��w�+*]gTw�څ�\'��\x7FZ�+3>\x17�ǌ\x0Fҏ���݇\x   x07\x18�Uk\r\x13\r^�77�9�\x00���5"k\x17�%o.q�eo��\x1E>��)˕�\x18c<\x0E���ٰ��� ��n��%<�U�tu��lz\x1C~4V���cvX�̯��o��4��\x00�5P��\x01\x05��\x1A�V�.�zDk�����\x00\x11�o+c�`z\x0E�\x1BjjX+�AA�\'\x07\x03�p^\x00,X�����?gF�F8\x18�N�\x00V V�åRFrc@\'\x1BA��6X�S�#?�5�q�8��5\x00\x12O"Ĭw�\x05\x03\x1C��r�(�����_e���e\'�=k���Z��m\x16�?���k��ހ-��\x04a�\x03޺�����k�U��;aCK����?�ą���8ʐs�V4֒B\x11�]��J�\r{��\x14�O\x13�*+*(�I�����(׼��R��ty�\x06�Q�GP;QV. k{��\x0B\x10;u��\x1CCRg%�T�V������\x00�j�\'\n' +
    "[$\x0E3V�L\r^�\x101�'��5T��\x1C\x1C�k\x1D\x06h۠�\t'���{g\x1F�<gi\n" +
    "F1�i�����<�<\x1A�c\x07\x03�J�n�5\x19G\f�\x1C�~5\x7FB�\x13�ڴ䪩�����UM�|\x1C��<WG��\x0B]��,!�\x00Ea�?��Φ�lޚ�h�=\x06{[�\ro2H\x01�\x13]\x0BB\x122��T\f��W+\r���fX��p�I\x1EF�8����/|���,r6�'�\\Z\x1D��\x19�׶��%V�\x1C�5x�^�(���+rX��Y��L�G\x19\x1BfL\x1F0\x1E�#�S��\x10\x17TUf�ֶ�\x18�D�)J�<�İ��@=���We'��Z��U�ĉ�¬`p3�?Z+��Zj�K��\x1A\x13��纨�\x00���\x1C����\x00}\x1A��˃�N3����u{�\x06|�#��1���c\x0F�\x00# n篧�X��2�~��\x00�MO���\x0EGj�M¤�O\x01��\x14\x03��GZ����5\n" +
    `���r�1�\x0FJ�<\x00\x00����\x03u���>i\x18#�\rt>\x0F�[mz6�K!\x03\x1D�*�ţz/T�S{i[U�,3,c�>�ү�A�\x15X\x10I�$Uk{��o\x04��\x06=y���#$�f�l\f�vH5Ė���%��,yv�\x1D+���b�\x0E���7J��S�9�\x0F^ԣ���~@#\x0B���0J>�슒�\x0B:���6�k�"c�*~�'�Q\\}ƥ.�t�ϵ8 <(���9�{h�kR�W�\x1Dq;���5:���Y��Z��u����.\x01�9=@$\x0Fƫjy]b�\x03?�|\x7F�F�ZY�D�\x12�|�[8�I�m\fK�d\x05��q���N��s��mi��u\x13���O<�tWc�T\t�fN��g>�r\x7F\x1A���P�N;҅��Q\x0F�P.YT\x00��'�֥���^�,S+����Ҩ��#����O\x18�3P��hFU� �\\.h��\x15\x1D\x11��\x0E�\x0E�f��W��Ќ涷,q�.#�0\x1D\x15k��\x05��졒\x10\x04�r��+b��U�U��.:\x1D�W#����\t˔���\x10�p�\n` +
    `9'=���-�o\x14\x13\\\t\x167s�\x05�8\x1DO�tv�$�)��f���\x1C-c���d�L��6�\x1C\x7F"?Z���5}lcUI��m�\x18�\r�o�Է&��RC��\x07��\x15���汫��5kݱ�7\x0Fϧ�j{X�ĥ�\x0Fs�\x1C��MU�\x1A��Gly�\x7F��Gv\n` +
    '\x0E_�\x06\rC�\f��\x1E\b�O!\fr\x16��b�̲\x10\x17߰�i�x;�\x12=M(@\x00�9�6\x01�6\x1C�r{\x03��@g\f\x1Fo+�~\x14�RNs�LR�\x04\x00@=�\x14��{\x0E��wg\x05�x)*\x06\x18����Tm�+�t\x0F\x14^�K�ƫ=�wyNO\x07�Oo�u��=�Ei4ۄn�e W,�M=\x0E�U��;\ \x19�Ȱ03^q�V6ۥD۹�pö:/׽3X����h�a�,x�����;\n' +
    "㘴�'vX���&��'{Ȋ�SV���\x1C��E*�6q�\x8A�O��?��\x00",```

**Comment by ganigeorgiev** [comment]
I'm not sure that I understand what do you mean. Image thumbnails are automatically generated on first access if the requested thumb size is specified in your collection field options.

File upload is supported only via `multipart/form-data` requests. When using the JS SDK, the request body must be [`FormData`](https://developer.mozilla.org/en-US/docs/Web/API/FormData) instance. The file value must be [`Blob`](https://developer.mozilla.org/en-US/docs/Web/API/Blob) or [`File`](https://developer.mozilla.org/en-US/docs/Web/API/File).

For a web example you can also check https://pocketbase.io/docs/files-handling/#uploading-files.

If the above doesn't help, please elaborate and a provide more details (ideally with a code sample) what are you trying to do.
